solidity/test/bridge/Bridge.Wallets.test.ts

```diff
@@ -4,8 +4,13 @@ import { SignerWithAddress } from "@nomiclabs/hardhat-ethers/signers"
 import chai, { expect } from "chai"
 import { smock } from "@defi-wonderland/smock"
 import type { FakeContract } from "@defi-wonderland/smock"
-import { ContractTransaction } from "ethers"
-import type { Bridge, BridgeStub, IWalletRegistry } from "../../typechain"
+import { ContractTransaction, Signer } from "ethers"
```

Comment:
```suggestion
import { ContractTransaction } from "ethers"
```

Reply:
Done in fa2b374ef9e5abbbe9730262f1d550f14886b8ca.